README.md

```diff
@@ -1,13 +1,48 @@
-# Mongo API Project
+# Mongo API Project  
+This API can be used e.g. to create exercise programs.
 
-Replace this readme with your own information about your project. 
+---
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## Available endpoints and queries
 
-## The problem
+### All exercises  
+/exercises
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+### Find exercise by id  
+/exercises/exerciseId
+
+### Search exercises by name and target muscle  
+#### examples  
+* how to find all deadlift variations:  
+/exercise?name=deadlift  
+
+* how to find all exercises with target muscle gluteus maximus:  
+/exercise?targetMuscle=gluteus 
+
+### All multi-joint exercises  
+/category/multi-joint
+
+### All single-joint exercises  
+/category/single-joint
```

Comment:
I really like that you put your API documentation here, where it's easy to find. Also, great job that you did a documentation 🌟